schemas/interface.ts

```diff
@@ -183,4 +183,74 @@ declare namespace Quizoot {
          */
         expected_output: string;
     }
+
+    /**
+     * A Quiz object represents a set of questions.
+     *
+     * @version 0.1.0
+     */
+    export interface Quiz {
+        /**
+         * Id of the quiz.
+         */
+        id: string;
+        /**
+         * The title (header) of the quiz.
+         * @minLength 1
+         * @maxLength 40
+         */
+        title: string;
+        /**
+         * A brief description of the quiz content, topic...
+         */
+        description: string;
+        /**
+         * Maximum score the user can get. Will be used to compute the final score: sum(question_i_score) / max_score.
+         * @default 0
+         * @minimum 0
+         */
+        max_score: number;
+        /**
+         * All the questions the quiz is made of.
+         */
+        questions: QuestionItem[]
+        /**
+         * List of optional authors or contributors of the quiz. See {@link Author}.
+         */
+        authors?: Author[];
+        /**
+         * List of optional tags associated with the quiz.
+         */
+        tags?: string[];
+    }
+
+    interface QuestionItem {
+        /**
+         * Id of a question referenced in the quiz.
+         */
+        question_id: string;
+        /**
+         * Id of the next question in the quiz chronology. Is null if it is the last question.
+         */
+        next_question_id: string | null;
+        /**
+         * Id of the previous question in the quiz chronology. Is null if it is the first question.
+         */
+        prev_question_id: string | null;
+    }
+
+    interface Author {
```

Comment:
Awesome. We can add the email. But I'd rather make first name and last name required. and pseudo not required..

Reply:
Good idea for the email !
I made first name and last name not required for people who don't want to give their real identity.
No need ?

Reply:
The make at least FirstName required. i want to avoid having trouble to display the author in the frontend